### PR TITLE
Added badge

### PR DESCRIPTION
**Pull request by finnif**
1. add: added badge.
2. update: update UIWidgets version (dev).

### CODE REVIEW

File: Assets/ConnectApp/Utils/CCommonUtils.cs [hunk 1]
```diff
@@ -1,8 +1,11 @@
+using System.Collections.Generic;
+using ConnectApp.Models.Model;
+using Unity.UIWidgets.foundation;
 using Unity.UIWidgets.widgets;
 using UnityEngine;
 
 namespace ConnectApp.Utils {
-    public static class CCommonUtils {
```
Comment (zhuxingwei) [inline Assets/ConnectApp/Utils/CCommonUtils.cs]: will this class be instantiated ?

File: Assets/ConnectApp/Redux/Actions/LoginAction.cs [hunk 2]
```diff
@@ -116,6 +118,7 @@ public static partial class Actions {
                     .Catch(error => {
                         CustomDialogUtils.hiddenCustomDialog();
                         dispatcher.dispatch(new LoginByWechatFailureAction());
+                        Debug.Log(message: error);
```
Comment (zhuxingwei) [inline Assets/ConnectApp/Redux/Actions/LoginAction.cs]: remove this in release mode?

File: Assets/ConnectApp/Redux/Reducers/Reducer.cs [hunk 3]
```diff
@@ -863,6 +863,24 @@ public static class AppReducer {
                     break;
                 }
 
+                case UserLicenseMapAction action: {
```
Comment (zhuxingwei) [inline Assets/ConnectApp/Redux/Reducers/Reducer.cs]: here we do only add/modify operations and won't remove the entry which is in the old userLiscenseMap but not in the new map. is it ok?

File: Assets/ConnectApp/Utils/CCommonUtils.cs (hunk 1 above)
Comment (finnif) [reply Assets/ConnectApp/Utils/CCommonUtils.cs]: fixed it

File: Assets/ConnectApp/Redux/Actions/LoginAction.cs (hunk 2 above)
Comment (finnif) [reply Assets/ConnectApp/Redux/Actions/LoginAction.cs]: fixed it

**Pull request merged**